sopel/plugins/rules.py

```diff
@@ -751,17 +750,17 @@ def get_global_metrics(self) -> RuleMetrics:
 
     @property
     @abc.abstractmethod
-    def user_rate_limit(self) -> datetime.timedelta:
+    def user_rate_limit(self) -> timedelta:
         """The rule's user rate limit."""
 
     @property
     @abc.abstractmethod
-    def channel_rate_limit(self) -> datetime.timedelta:
+    def channel_rate_limit(self) -> timedelta:
         """The rule's channel rate limit."""
 
     @property
     @abc.abstractmethod
-    def global_rate_limit(self) -> datetime.timedelta:
+    def global_rate_limit(self) -> timedelta:
         """The rule's global rate limit."""
```

Comment:
These are fixed in a commit with a typo in its message ("oudated"), which I can fix during final rebase.

(Please leave this discussion thread unresolved; I will use it as a reminder.)